userparameter_diskstats.conf

```diff
@@ -1,14 +1,13 @@
+UserParameter=custom.vfs.discover_disks,/usr/local/bin/lld-disks.py
```

Comment:
May we save original place of the script, as other users could already have it in it?